samples/bot-formatting-cards/nodejs/Readme.md

```diff
@@ -0,0 +1,131 @@
+---
+page_type: sample
+description: This sample demonstrates the use of different formatting on cards
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "01/28/2023 05:00:17 PM"
+urlFragment: officedev-microsoft-teams-samples-bot-formatting-cards-nodejs
+
+---
+# Different Formatting Cards
+
+This sample shows the feature where user can send different formatting on cards using bot.
```

Comment:
This sample shows the feature where user can use different formatting on adaptive cards using bot.

Reply:
Fixed